src/test/java/tictactoe/GameSaverTest.java

```diff
@@ -0,0 +1,81 @@
+package tictactoe;
+
+import org.junit.jupiter.api.Test;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class GameSaverTest {
+    Database database = new Database();
+    Connection connection = database.connect();
+    GameSaver gameSaver = new GameSaver(connection);
+
+    void clearTable() {
```

Comment:
What do you think about passing table name as an argument to `clearTable`?

Reply:
I tried doing this and adding it to the query with string interpolation, however I read doing this is a SQL injection vulnerability (not sure if that's right or not). Do you have any suggestions for doing this? @kotauror